### ISSUE TITLE
Support LocalTime

### ISSUE THREAD

**Issue by akuleshov7** [akuleshov7/ktoml#120]
LocalTime should be available if https://github.com/Kotlin/kotlinx-datetime/issues/57 is fixed, I may do a new PR once a local time implementation lands in kotlinx-datetime

_Originally posted by @bishiboosh in https://github.com/akuleshov7/ktoml/issues/119#issuecomment-1054144433_

**Comment by akuleshov7** [comment]
Also - need to add some examples of decoding with Date to the Example section.

**Comment by NightEule5** [comment]
The PR has apparently merged now as part of 0.4.0

**Comment by akuleshov7** [comment]
> The PR has apparently merged now as part of 0.4.0

Where is it? I do not see…

**Comment by NightEule5** [comment]
https://github.com/Kotlin/kotlinx-datetime/releases/tag/v0.4.0

**Comment by akuleshov7** [comment]
Ah, I got it! Thank you, sounds good, we will be able just to update it
